kousa/test/broth/_message/user/admin_update_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule BrothTest.Message.User.AdminUpdateTest do
+  use ExUnit.Case, async: true
+  use KousaTest.Support.EctoSandbox
+
+  alias Beef.Schemas.User
+  alias Beef.Users
+  alias BrothTest.WsClient
+  alias BrothTest.WsClientFactory
+  alias KousaTest.Support.Factory
+
+  require WsClient
+
+  setup do
+    user = Factory.create(User)
+    client_ws = WsClientFactory.create_client_for(user)
```

Comment:
This is not the right type of test, you shouldn't be making ws calls here. All ws calls should be kept in `kousa/test/broth/user/admin_update_test.exs`.

Look at `kousa/test/broth/_message/user/update_test.exs` for an example

Reply:
fixed it